src/middlewared/middlewared/plugins/truecommand/wireguard.py

```diff
@@ -99,7 +113,8 @@ async def connected(self, app):
         connected = Status(tc_config['status']) == Status.CONNECTED
         return {
             'connected': connected,
-            'truecommand_ip': tc_config['remote_address'] if connected else None,
+            'truecommand_ip': tc_config['remote_ip_address'] if connected else None,
+            'truecommand_address': tc_config['remote_address'] if connected else None,
```

Comment:
On second thought, dont you think address is confusing?

Address can refer to many things. IP Address, URL Address.
How about we use IP and URL to lessen the confusion?

Reply:
Right again ; )